core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/config/ShaclSailSchema.java

```diff
@@ -55,7 +55,7 @@
 	/** <code>http://rdf4j.org/config/sail/shacl#serializableValidation</code> */
 	public final static IRI SERIALIZABLE_VALIDATION = create("serializableValidation");
 
-	public final static IRI SHACL_ADVANCED_FEATURES = create("shaclAdvancedFeatures");
+	public final static IRI ECLIPSE_RDF4J_SHACL_EXTENSIONS = create("eclipseRdf4jShaclExtensions");
```

Comment:
Perhaps leave off the Eclipse bit, just call it `RDF4J_SHACL_EXTENSION`?  Same for the actual getter/setter methods. Also fine if you prefer the longer version.

Reply:
I prefer the legitimacy of having it under eclipse rather than just RDF4J.